sdk/cargo-build-bpf/src/main.rs

```diff
@@ -142,26 +250,89 @@ fn build_bpf_package(config: &Config, target_directory: &Path, package: &cargo_m
         println!("Legacy program feature detected");
     }
 
-    let xargo_build = config.bpf_sdk.join("rust").join("xargo-build.sh");
-    let mut xargo_build_args = vec![];
+    let criterion_version = if cfg!(target_os = "macos") {
```

Comment:
Can we remove the Criterion dependency entirely here?  It's only used for C right?

Reply:
I don't know. If it's not needed, sure, easy to remove.

Reply:
removed